tests/views/playingScreen-test.js

```diff
@@ -9,39 +9,46 @@ var PlayingScreen = require('../../js/views/playingScreen');
 var UnmuteIcon = require('../../js/components/unmuteIcon');
 
 describe('PlayingScreen', function () {
+  var mockController, closedCaptionOptions;
   var handleVrPlayerMouseUp = function() {};
-  it('creates a PlayingScreen and checks mouseMove, mouseUp without video360', function () {
-    var isMoved = false
-      , isTouched = false;
-    var mockController = {
-      videoVr: false,
+
+  beforeEach(function() {
+    mockController = {
       state: {
         isMobile: false,
         accessibilityControlsEnabled: false,
+        controlBarVisible: true,
         upNextInfo: {
           showing: false
         },
         volumeState: {
-          muted: false
+          muted: false,
+          mutingForAutoplay: false
         },
         config: {
           isVrAnimationEnabled: {
             vrNotification: true,
             vrIcon: true
-          },
+          }
         }
-      },
-      startHideControlBarTimer: function() {
-        isMoved = true;
-      },
-      onTouched: function() {
-        isTouched = true;
       }
     };
-
-    var closedCaptionOptions = {
+    closedCaptionOptions = {
       cueText: "cue text"
     };
+  });
+
+  it('creates a PlayingScreen and checks mouseMove, mouseUp without video360', function () {
+    var isMoved = false
+      , isTouched = false;
```

Comment:
Should we be checking this variable in this unit test? It looks to be unused right now.

Reply:
Not sure, these are mostly old unit tests which I refactored in order to use `mockController` instead of declaring one each time, but I I'll check and remove if necessary.

Reply:
@aeng7 so I think this unit test is wrong. It looks like it might be testing that these events are not fired for regular videos, however, this is not tested properly because the event listeners are being added to the `document`, so they are not triggered by `TestUtils.Simulate`. If I fix this in order to use DOM events the unit test fails because the events are actually being triggered even without 360 video. I'm not sure if it would be better to just remove this.

Reply:
@jpfranco should we have EPAM look at this unit test?

Reply:
@aeng7 yeah I think that would probably be better. I might be misunderstanding what this does.